Dockerfile

```diff
@@ -1,13 +1,22 @@
-FROM google/debian:wheezy
+FROM gcr.io/google_appengine/base
+
+# prepare the image
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && apt-get install -y -qq --no-install-recommends wget unzip python php5-mysql php5-cli php5-cgi openjdk-7-jre-headless openssh-client python-openssl && apt-get clean
-RUN wget https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.zip && unzip google-cloud-sdk.zip && rm google-cloud-sdk.zip
+
+# install the Google Cloud SDK
+ENV HOME /
 ENV CLOUDSDK_PYTHON_SITEPACKAGES 1
-RUN google-cloud-sdk/install.sh --usage-reporting=true --path-update=true --bash-completion=true --rc-path=/.bashrc --disable-installation-options
-RUN google-cloud-sdk/bin/gcloud --quiet components update pkg-go pkg-python pkg-java preview alpha beta app
-RUN google-cloud-sdk/bin/gcloud --quiet config set component_manager/disable_update_check true
+RUN wget https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.zip && unzip google-cloud-sdk.zip && rm google-cloud-sdk.zip
+RUN google-cloud-sdk/install.sh --usage-reporting=true --path-update=true --bash-completion=true --rc-path=/.bashrc --additional-components app alpha beta
+
+# disable updater check for the whole installation
+RUN google-cloud-sdk/bin/gcloud config set --installation component_manager/disable_update_check true
+
+# disable updater completely
+RUN sed -i -- 's/\"disable_updater\": false/\"disable_updater\": true/g' /google-cloud-sdk/lib/googlecloudsdk/core/config.json
```

Comment:
Because that just disables the automatic check if a new version of gcloud is available. People can still do `gcloud components update`, let it actually download/ finish installing, and then wonder why it didn't do anything. :)